diff --git a/etc/init/keystore2.rc b/etc/init/keystore2.rc
--- a/etc/init/keystore2.rc
+++ b/etc/init/keystore2.rc
@@ -6,12 +6,10 @@
 #
 # See system/core/init/README.md for information on the init.rc language.
 
-on late-init
-    start keystore2
-
 service keystore2 /system/bin/keystore2 /tmp/misc/keystore
     class early_hal
     user root
     group keystore readproc log
     writepid /dev/cpuset/foreground/tasks
+    disabled
     seclabel u:r:recovery:s0
