diff --git a/Android.mk b/Android.mk
index d334f33..5c34b89 100644
--- a/Android.mk
+++ b/Android.mk
@@ -397,7 +397,7 @@ ifneq ($(TW_CUSTOM_CPU_TEMP_PATH),)
 	LOCAL_CFLAGS += -DTW_CUSTOM_CPU_TEMP_PATH=$(TW_CUSTOM_CPU_TEMP_PATH)
 endif
-ifneq ($(TW_EXCLUDE_ENCRYPTED_BACKUPS),)
+ifneq ($(TW_EXCLUDE_ENCRYPTED_BACKUPS),true)
     LOCAL_SHARED_LIBRARIES += libopenaes
 else
     LOCAL_CFLAGS += -DTW_EXCLUDE_ENCRYPTED_BACKUPS
 endif
@@ -588,7 +588,7 @@ endif
 ifeq ($(BOARD_HAS_NO_REAL_SDCARD),)
     TWRP_REQUIRED_MODULES += sgdisk
 endif
-ifneq ($(TW_EXCLUDE_ENCRYPTED_BACKUPS),)
+ifneq ($(TW_EXCLUDE_ENCRYPTED_BACKUPS),true)
     TWRP_REQUIRED_MODULES += openaes openaes_license
 endif
 ifeq ($(TW_INCLUDE_FB2PNG), true)
diff --git a/prebuilt/Android.mk b/prebuilt/Android.mk
index ee02252..dcbcd73 100755
--- a/prebuilt/Android.mk
+++ b/prebuilt/Android.mk
@@ -310,6 +310,6 @@ ifneq ($(wildcard system/core/libsparse/Android.*),)
     RECOVERY_LIBRARY_SOURCE_FILES += $(TARGET_OUT_SHARED_LIBRARIES)/libsparse.so
 endif
-ifneq ($(TW_EXCLUDE_ENCRYPTED_BACKUPS),)
+ifneq ($(TW_EXCLUDE_ENCRYPTED_BACKUPS),true)
     RECOVERY_LIBRARY_SOURCE_FILES += $(TARGET_OUT_SHARED_LIBRARIES)/libopenaes.so
 endif
 ifeq ($(TARGET_USERIMAGES_USE_F2FS), true)
@@ -485,6 +485,6 @@ include $(BUILD_PHONY_PACKAGE)
 
 # copy license file for OpenAES
-ifneq ($(TW_EXCLUDE_ENCRYPTED_BACKUPS),)
+ifneq ($(TW_EXCLUDE_ENCRYPTED_BACKUPS),true)
     include $(CLEAR_VARS)
     LOCAL_MODULE := openaes_license
     LOCAL_MODULE_TAGS := optional
